frontend/pulse3d/src/pages/uploads.js

```diff
@@ -175,12 +307,13 @@ export default function Uploads() {
   useEffect(() => {
     getAllJobs();
     // start 10 second interval
+    /*
```

Comment:
I think  we want to keep this uploadsInterval.. I'm not sure why the initial state is also commented out. it's being used in the RC10132022 branch

Reply:
Could you actually merge the RC10132022 branch into this one before merging? it's missing a few changes and just want to make sure they don't get overridden

Reply:
I think I already merged it. I can fix this issue manually As well. Il make sure to keep the updates for job status